module.js

```diff
@@ -15,6 +15,7 @@ var hook = function(req, res) {
 
 var github = function(payload, localToken) {
     // TODO: Only acknowledge pushes to the "Master" branch.
+    console.log("Repo: "+payload.repository.full_name);
```

Comment:
It does, however, since there are several merges that were made without beautification, I was unable to apply it to this merge without breaking the stylus support update.  We really need to apply a cross the board beautification to keep from having this problem.  Once the stylus PR is merged, I'll revase this one and rebeautify.